tests/testUtils.py

```diff
@@ -1611,14 +1611,18 @@ def relaunchEosInstances(self):
 
         return self.updateNodesStatus()
 
+    def dumpErrorDEtailImpl(self,fileName):
```

Comment:
`dumpErrorDetailsImpl` typo of function name, this one I assume is related to class `net_plugin_impl` in `net_plugin.cpp`